codenames/gameplay/ai2_hack.py

```diff
@@ -86,18 +86,49 @@ def _apply_guess(self, guess):
             self.cumulative_score += guess_reward
             return guess_reward
 
-    def apply_guesses(self, clue: Clue, guessed_words: List[str]):
+    # This method executes the guesses of team1.
+    def apply_team1_guesses(self, clue: Clue, guessed_words: List[str]):
         guess_list_rewards = []
         if len(guessed_words) > int(clue.count) + 1:
             raise Exception
         for word in guessed_words:
-            guess_reward = self._apply_guess(word)
+            guess_reward = self._apply_team1_guess(word)
             guess_list_rewards.append(guess_reward)
             if guess_reward <= 0:
                 break
 
+        # team1 just played, so we need to update the internal state of
+        # the engine pertaining to turn info.
+        self.engine.next_turn()
+
         return guess_list_rewards
 
+    # The engine was designed for a 2-player experience. In the 1-player
+    # version of the game, there is an imaginary team2 which is pretty
+    # boring: it reveals one of the team2 cards in each turn. This method
+    # simulates the imaginary boring team2 for the 1-player version of the 
+    # game.
+    def apply_team2_guesses(self):
+        team2_guess = None
+        for i in range(len(self.engine.board)):
+            # find the first word which belongs to team2 and is not revealed.
+            if self.engine.owner[i] == BAD and self.engine.assignment_not_revealed[i]:
```

Comment:
Does it make sense to rename `GOOD` and `BAD` to `player_1_s_card` and `player_2_s_card` ?